Gems/Terrain/Code/Source/TerrainRenderer/TerrainMacroMaterialManager.h

```diff
@@ -43,7 +40,7 @@ namespace Terrain
 
     private:
 
-        static constexpr auto InvalidImageIndex = AZ::Render::BindlessImageArrayHandler::InvalidImageIndex;
+        static constexpr auto InvalidImageIndex = AZStd::numeric_limits<uint32_t>::max();
```

Comment:
The named constant seemed better, why switch to manually calculating it?

Reply:
nm, The change is more obvious in the side by side view.

Reply:
It was just "named" because it was referring to a different place where it was actually defined, but `BindlessImageArrayHandler` no longer exists. I think using limits here is fine.